src/file/file.service.ts

```diff
@@ -176,13 +257,14 @@ export class FileService {
   static async updateMany(idList: string[], partialFile: Partial<IFile>): Promise<{ id: string, error: Error }[]> {
     const extractedPF: Partial<IFile> = this.extractQuery(partialFile);
     const failedFiles: { id: string, error: Error }[] = [];
-    for (let i = 0; i < idList.length; i++) {
+
+    await Promise.all(idList.map(async (id) => {
```

Comment:
Nice

Reply:
if its the empty line that you're annoyed about, i removed it.

Reply:
again cant understand what you want because you dont explain.
is it the empty line about the promise.all or the promise.all itself?